go/Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM ubuntu:14.04
+FROM alpine:3.3
 MAINTAINER Keybase <admin@keybase.io>
 
-RUN apt-get update && apt-get install -y fuse openssl jq
+RUN apk add --update fuse jq bash && \
+    rm -rf /var/cache/apk/* && \
+    mkdir -p /home/keybase && \
+    mkdir /keybase && \
+    adduser -D keybase && \
+    chown keybase:keybase /keybase
```

Comment:
In our Linux install scripts we leave this folder owned by root, but we `chmod 777` it. Might be good to stick with that just so that we test the same setup we're shipping?